lib/miq_automation_engine/service_models/miq_ae_service_iso_datastore.rb

```diff
@@ -0,0 +1,5 @@
+module MiqAeMethodService
+  class MiqAeServiceIsoDatastore < MiqAeServiceModelBase
+    expose :iso_images, :association => true
```

Comment:
@matobet Looks good. I was also wondering about exposing the ext_management_system as @gmcculloug mentioned.